### PR TITLE
Rename 'Weight' field in question form to 'Order'.

### LINKED ISSUE #1611: Rename question weight field
Change display title of the field to `Order`. This is a temporary fix in response to user feedback.

It will be superseded by drag and drop reordering, hiding the `Weight` field from the end user.

### PR DESCRIPTION
**Pull request by fonglh**
Fixes #1611.

In response to user feedback. Will be made obsolete by drag and drop reordering.

### CODE REVIEW

File: app/views/course/assessment/questions/_form.html.slim [hunk 1]
```diff
@@ -2,5 +2,5 @@
 = f.input :description
 = f.input :staff_only_comments
 = f.input :maximum_grade
-= f.input :weight
+= f.input :weight, label: I18n.t('course.assessment.questions.form.weight')
```
Comment (allenwq) [inline app/views/course/assessment/questions/_form.html.slim]: Use human attributes

**Comment by allenwq** [comment]
There are categories and tabs which also have weights, rename them as well ?

**Comment by fonglh** [comment]
@allenwq done! Also made the same rename for assessment categories and tabs.

**Comment by allenwq** [comment]
👍

**Pull request merged**